solana/utils/ed25519_base.py

```diff
@@ -0,0 +1,50 @@
+Q = 2 ** 255 - 19
```

Comment:
Could you site where you got this file from? i.e. Add link to original source.

Reply:
https://github.com/warner/python-pure25519/blob/master/pure25519/basic.py#L132